### ISSUE TITLE
Empty query responses throwing WrongValueException

### ISSUE THREAD

**Issue by tez-m** [fsanaulla/chronicler#185]
Following examples throw WrongValueException:

```
    val influxMng = InfluxMng("localhost", 8086, None)
    val create = influxMng.createDatabase("test_database")
    Await.result(create, 10.seconds) match {
      case Left(e) => throw e
      case Right(result) => println(result)
    }

    // Empty series
    val influxIo = InfluxIO("localhost", 8086, None, false)
    val query = influxIo.database("test_database").readJson("SELECT * FROM some_table")
    Await.result(query, 10.seconds) match {
      case Left(e) => throw e
      case Right(result) => println(result)
    }
```

```
    val influxMng = InfluxMng("localhost", 8086, None)
    val create = influxMng.createDatabase("test_database")
    Await.result(create, 10.seconds) match {
      case Left(e) => throw e
      case Right(result) => println(result)
    }

    val influxIo = InfluxIO("localhost", 8086, None, false)
    val point = Point("some_table")
      .addTag("some_tag", "tag 123")
      .addField("some_value", 1)
    val write = influxIo.database("test_database").writePoint(point)
    Await.result(write, 10.seconds) match {
      case Left(e) => throw e
      case Right(result) => println(result)
    }

    // Empty result
    val query = influxIo.database("test_database").readJson("SELECT * FROM some_table WHERE time > now()")
    Await.result(query, 10.seconds) match {
      case Left(e) => throw e
      case Right(result) => println(result)
    }
```

Exception thrown:

```
expected array, got null
org.typelevel.jawn.ast.WrongValueException: expected array, got null
	at com.github.fsanaulla.chronicler.core.jawn.package$RichJValue$.arrayValue$extension(package.scala:43)
	at com.github.fsanaulla.chronicler.core.components.JsonHandler$$anonfun$queryResult$2.apply(JsonHandler.scala:111)
	at com.github.fsanaulla.chronicler.core.components.JsonHandler$$anonfun$queryResult$2.apply(JsonHandler.scala:112)
	at scala.util.Either$RightProjection.flatMap(Either.scala:522)
	at com.github.fsanaulla.chronicler.core.either.package$EitherOps$.flatMapRight$extension(package.scala:49)
	at com.github.fsanaulla.chronicler.core.components.JsonHandler.queryResult(JsonHandler.scala:112)
	at com.github.fsanaulla.chronicler.core.components.ResponseHandler$$anonfun$queryResultJson$1$$anonfun$apply$4.apply(ResponseHandler.scala:79)
	at com.github.fsanaulla.chronicler.core.components.ResponseHandler$$anonfun$queryResultJson$1$$anonfun$apply$4.apply(ResponseHandler.scala:79)
	at scala.util.Either$RightProjection.flatMap(Either.scala:522)
	at com.github.fsanaulla.chronicler.core.either.package$EitherOps$.flatMapRight$extension(package.scala:49)
	at com.github.fsanaulla.chronicler.core.components.ResponseHandler$$anonfun$queryResultJson$1.apply(ResponseHandler.scala:79)
	at com.github.fsanaulla.chronicler.core.components.ResponseHandler$$anonfun$queryResultJson$1.apply(ResponseHandler.scala:78)
	at com.github.fsanaulla.chronicler.core.implicits.package$$anon$1.map(package.scala:27)
	at com.github.fsanaulla.chronicler.core.components.ResponseHandler.queryResultJson(ResponseHandler.scala:78)
	at com.github.fsanaulla.chronicler.core.api.DatabaseApi$$anonfun$readJson$1.apply(DatabaseApi.scala:115)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:253)
	at scala.concurrent.Future$$anonfun$flatMap$1.apply(Future.scala:251)
	at scala.concurrent.impl.CallbackRunnable.run(Promise.scala:36)
	at scala.concurrent.impl.ExecutionContextImpl$AdaptedForkJoinTask.exec(ExecutionContextImpl.scala:121)
	at scala.concurrent.forkjoin.ForkJoinTask.doExec(ForkJoinTask.java:260)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.pollAndExecAll(ForkJoinPool.java:1253)
	at scala.concurrent.forkjoin.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1346)
	at scala.concurrent.forkjoin.ForkJoinPool.runWorker(ForkJoinPool.java:1979)
	at scala.concurrent.forkjoin.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:107)
```

**Comment by fsanaulla** [comment]
Looks like decoding issue will be fixed in 0.5.6

**Comment by fsanaulla** [comment]
Solved in #187
